trk/src/Screens/TabScreens/GymAnalytics/GymDaily/Backend/analyticsCalculations.js

```diff
@@ -66,17 +70,34 @@ export const fetchTapsAndCalculateAscents = async (yourClimbs) => {
   let peakTimeString = peakHour ? `${peakHour}:00 - ${parseInt(peakHour) + 1}:00` : '';
 
   // Determine most and least completed climbs
-  let mostCompletedClimb = Object.values(climbFrequencyMap).reduce((a, b) => (a && b && a.count > b.count) ? a : b, null);
-  let leastCompletedClimb = Object.values(climbFrequencyMap).reduce((a, b) => (a && b && a.count < b.count) ? a : b, null);
-
+  let mostCompletedClimbs = [];
+  let leastCompletedClimbs = [];
+  let maxCount = 0;
+  let minCount = Infinity;
+  
+  Object.values(climbFrequencyMap).forEach(climb => {
+    if (climb.count > maxCount) {
+      mostCompletedClimbs = [climb];
+      maxCount = climb.count;
+    } else if (climb.count === maxCount) {
+      mostCompletedClimbs.push(climb);
+    }
+  
+    if (climb.count < minCount) {
+      leastCompletedClimbs = [climb];
+      minCount = climb.count;
+    } else if (climb.count === minCount) {
+      leastCompletedClimbs.push(climb);
+    }
+  });
   // Return the calculated values
   return {
     totalTaps,
     allTaps,
     peakTimeString,
     latestAscentsCount,
-    mostCompletedClimb,
-    leastCompletedClimb
+    mostCompletedClimbs,
+    leastCompletedClimbs
   };
 
 };
```

Comment:
these are now plural since they are arrays that can contain multiple climbs (if the lowest and highest counts are held by more than 1 climb)